site2/docs/tiered-storage-filesystem.md

```diff
@@ -392,13 +392,13 @@ Execute the following commands in the repository where you download Pulsar tarba
 1. Start Pulsar standalone.
 
     ```
-    ./bin/pulsar standalone -a 127.0.0.1
+    bin/pulsar standalone -a 127.0.0.1
```

Comment:
I think the original text is correct, why modify this?

Reply:
Yes, `.bin/pulsar` also works, i just want to keep consistent with other docs like [administration-geo](https://pulsar.apache.org/docs/en/administration-geo/) and [standalone consume-a-message)](https://pulsar.apache.org/docs/en/standalone/#consume-a-message). Most of Pulsar docs use `bin/pulsar`.